commit 4ee0bd3c8ead89541ab7d196fb54e940e397420d
Author: Oliver Kurth <okurth@vmware.com>
Date:   Fri Feb 7 10:39:38 2020 -0800

    Rectify a log spew in vmsvc logging (vmware-vmsvc-root.log )
    
    When a LSI Logic Parallel SCSI controller sits in PCI bus 0
    (SCSI controller 0), the Linux disk device enumeration does not provide
    a "label" file with the controller name.  This results in messages like
    
    "GuestInfoGetDiskDevice: Missing disk device name; VMDK mapping unavailable for "/var/log", fsName: "/dev/sda2"
    
    repeatedly appearing in the vmsvc logging.  The patch converts what previously
    was a warning message to a debug message and thus avoids the log spew.

diff --git a/open-vm-tools/services/plugins/guestInfo/diskInfo.c b/open-vm-tools/services/plugins/guestInfo/diskInfo.c
index 878276ce..032c0928 100644
--- a/open-vm-tools/services/plugins/guestInfo/diskInfo.c
+++ b/open-vm-tools/services/plugins/guestInfo/diskInfo.c
@@ -846,9 +846,9 @@ GuestInfoGetDiskDevice(const char *fsName,
     */
    for (indx = 0; indx < partEntry->diskDevCnt; indx++) {
       if (partEntry->diskDevNames[indx][0] == '\0') {
-         g_warning("%s: Missing disk device name; VMDK mapping unavailable "
-                   "for \"%s\", fsName: \"%s\"\n", __FUNCTION__,
-                   partEntry->name, fsName);
+         g_debug("%s: Missing disk device name; VMDK mapping unavailable "
+                 "for \"%s\", fsName: \"%s\"\n", __FUNCTION__,
+                 partEntry->name, fsName);
          partEntry->diskDevCnt = 0;
          free(partEntry->diskDevNames);
          partEntry->diskDevNames = NULL;
